fortnite_bot.py

```diff
@@ -257,40 +257,68 @@ async def replays_operations(ctx, *params):
         await ctx.send("No replay file found")
         return
 
-    command = params.pop(0) if params else None
+    command = None
+    username = None
+    if len(params) == 2:
+        username = params.pop(1)
+        command = params.pop(0) 
+        if username not in SQUAD_PLAYERS_LIST:
+            await ctx.send(f"{username} provided is not a valid squad player")
```

Comment:
Do you want to return after this?

Reply:
oh yeah ur right, should I just `return`